include/settings/settings.h

```diff
@@ -193,33 +181,6 @@ int settings_get_value(char *name, char *buf, int buf_len);
  */
 int settings_commit(char *name);
 
```

Comment:
No, this is not possible because it calls settings_parse and lookup that changes name, could be done with a local name but I prefer to keep as it was.

Reply:
You are right!